### PR TITLE
[zh] Sync changes from English site (6)

### PR DESCRIPTION
**Pull request by tengqm**

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/replicaset.md [hunk 1]
```diff
@@ -441,11 +627,25 @@ machine-level function, such as machine monitoring or machine logging.  These po
...
 
+### ReplicationController
 
+<!--
+ReplicaSets are the successors to [_ReplicationControllers_](/docs/concepts/workloads/controllers/replicationcontroller/).
+The two serve the same purpose, and behave similarly, except that a ReplicationController does not support set-based
+selector requirements as described in the [labels user guide](/docs/concepts/overview/working-with-objects/labels/#label-selectors).
+As such, ReplicaSets are preferred over ReplicationControllers
+-->
+ReplicaSets 是 [ReplicationControllers](/zh/docs/concepts/workloads/controllers/replicationcontroller/)
```
Comment (sftim) [inline content/zh/docs/concepts/workloads/controllers/replicaset.md:646]:
```suggestion
ReplicaSet 是 [ReplicationController](/zh/docs/concepts/workloads/controllers/replicationcontroller/)
```
?

File: content/zh/docs/concepts/workloads/controllers/replicaset.md [hunk 2]
```diff
@@ -51,41 +55,14 @@ ReplicaSet 所获得的 Pod 都在其 ownerReferences 字段中包含了属主 R
...
-A ReplicaSet identifies new Pods to acquire by using its selector. If there is
-a Pod that has no OwnerReference or the OwnerReference is not a {{<
-glossary_tooltip term_id="controller" >}} and it matches a ReplicaSet's
-selector, it will be immediately acquired by said ReplicaSet.
+A ReplicaSet identifies new Pods to acquire by using its selector. If there is a Pod that has no OwnerReference or the
+OwnerReference is not a {{< glossary_tooltip term_id="controller" >}} and it matches a ReplicaSet's selector, it will be immediately acquired by said ReplicaSet.
 -->
 ReplicaSet 使用其选择算符来辨识要获得的 Pod 集合。如果某个 Pod 没有
 OwnerReference 或者其 OwnerReference 不是一个 
 {{< glossary_tooltip text="控制器" term_id="controller" >}}，且其匹配到
```
Comment (sftim) [inline content/zh/docs/concepts/workloads/controllers/replicaset.md:63]:
```suggestion
{{< glossary_tooltip text="控制器" term_id="controller" >}}、且其匹配到
```
?

File: content/zh/docs/concepts/workloads/controllers/replicaset.md [hunk 3]
```diff
@@ -118,151 +97,349 @@ ReplicaSet 确保任何时间都有指定数量的 Pod 副本在运行。
...
+For ReplicaSets, the kind is always just ReplicaSet.
+In Kubernetes 1.9 the API version `apps/v1` on the ReplicaSet kind is the current version and is enabled by default. The API version `apps/v1beta2` is deprecated.
+Refer to the first lines of the `frontend.yaml` example for guidance.
+
+The name of a ReplicaSet object must be a valid
+[DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
+
+A ReplicaSet also needs a [`.spec` section](https://git.k8s.io/community/contributors/devel/api-conventions.md#spec-and-status).
 -->
+## 编写 ReplicaSet 的 Spec
```
Comment (sftim) [inline content/zh/docs/concepts/workloads/controllers/replicaset.md:357]:
```suggestion
## 编写 ReplicaSet 的 spec
```
spec comes from `.spec` in the API so I would keep it all-lowercase.

**Review by sftim — COMMENTED** [review]
Three tiny tweaks. All optional.

**Comment by howieyuen** [comment]
/lgtm
you have contributed a lot recently, thanks for the contribution.

**Comment by tengqm** [comment]
> you have contributed a lot recently, thanks for the contribution.

That is because 1.19 is to be archived in a few days ... getting things fixed for 1.19 would be much difficult.

**Pull request merged**